L2SIVacuum/POUs/Functions/Gauges/FB_MKS422.TcPOU

```diff
@@ -54,15 +52,18 @@ END_IF
 	
 
 (* Pressure gauge State checks *)
-IF (rV <=9.6 ) AND (rV>=0.6) THEN
+IF (rV <=9.6 ) AND (rV>=0.6) AND (IG.xHV_SW) THEN
 	IG.eState := Valid; // normal
-ELSIF rV >= 0.18 AND rV <= 0.22 THEN 
+ELSIF rV >= 0.18 AND rV <= 0.22 AND (IG.xHV_SW) THEN 
 	IG.eState := ValidLo; //LO
-ELSIF rV > 9.6 AND rV<= 9.9 THEN
+ELSIF rV > 9.6 AND rV<= 9.9 AND (IG.xHV_SW) THEN
 	IG.eState := ValidHi; //HIGH
 ELSIF rV < 0.18 THEN // 
```

Comment:
Probably a dumb question as I don't , but should this include: `AND Ig.xHV_SW` like the others?
(If so, it might be a bit more readable to do a top-level `IF xHV_SW THEN (check voltages) ELSE set_off_state`)

Reply:
is it missing the range rv< 0.6 and rv>0.22? which should be valid or valid low?

Reply:
Should these values be local constants? Avoiding hardcoded values?